### PR TITLE
Fabo/add vesting warning

### PR DESCRIPTION
**Pull request by faboweb**
Closes #ISSUE

**Description:**

image

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: app/src/renderer/components/wallet/SendModal.vue [hunk 1]
```diff
@@ -115,7 +121,8 @@ export default {
     address: ``,
     amount: null,
     denom: ``,
-    num
+    num,
+    vestedWarning: `has vested coins. You will not be able to send those until the vesting period is over.`
```
Comment (jbibla) [inline app/src/renderer/components/wallet/SendModal.vue]:
```suggestion
    vestedWarning: `has vesting coins. You will not be able to send these until the vesting period is over.`
```

File: app/src/renderer/connectors/api.js [hunk 2]
```diff
@@ -53,7 +53,9 @@ const Client = (axios, remoteLcdURL) => {
               return emptyAccount
             }
             account = Object.assign(
-              {},
+              {
+                vested: true
```
Comment (jbibla) [inline app/src/renderer/connectors/api.js]:
```suggestion
                vesting: true
```

File: app/src/renderer/vuex/modules/wallet.js [hunk 3]
```diff
@@ -12,7 +12,8 @@ export default ({ node }) => {
     accountNumber: null,
     address: null,
     subscribedRPC: null,
-    externals: { config }
+    externals: { config },
+    vestedAccount: false
```
Comment (jbibla) [inline app/src/renderer/vuex/modules/wallet.js]:
```suggestion
    vestingAccount: false
```

File: app/src/renderer/vuex/modules/wallet.js [hunk 4]
```diff
@@ -37,6 +38,9 @@ export default ({ node }) => {
     },
     setAccountNumber(state, accountNumber) {
       state.accountNumber = accountNumber
+    },
+    setVestedAccount(state) {
```
Comment (jbibla) [inline app/src/renderer/vuex/modules/wallet.js]:
```suggestion
    setVestingAccount(state) {
```

File: app/src/renderer/vuex/modules/wallet.js [hunk 5]
```diff
@@ -37,6 +38,9 @@ export default ({ node }) => {
...
+      state.vestedAccount = true
```
Comment (jbibla) [inline app/src/renderer/vuex/modules/wallet.js]:
```suggestion
      state.vestingAccount = true
```

File: app/src/renderer/vuex/modules/wallet.js [hunk 6]
```diff
@@ -72,10 +76,13 @@ export default ({ node }) => {
       try {
         const res = await node.getAccount(state.address)
         state.error = null
-        const { coins, sequence, account_number } = res || {}
+        const { coins, sequence, account_number, vested } = res || {}
```
Comment (jbibla) [inline app/src/renderer/vuex/modules/wallet.js]:
```suggestion
        const { coins, sequence, account_number, vesting } = res || {}
```

File: app/src/renderer/vuex/modules/wallet.js [hunk 7]
```diff
@@ -72,10 +76,13 @@ export default ({ node }) => {
...
         commit(`setNonce`, sequence)
         commit(`setAccountNumber`, account_number)
         commit(`setWalletBalances`, coins || [])
+        if (vested) {
```
Comment (jbibla) [inline app/src/renderer/vuex/modules/wallet.js]:
```suggestion
        if (vesting) {
```

File: app/src/renderer/vuex/modules/wallet.js [hunk 8]
```diff
@@ -72,10 +76,13 @@ export default ({ node }) => {
...
+          commit(`setVestedAccount`)
```
Comment (jbibla) [inline app/src/renderer/vuex/modules/wallet.js]:
```suggestion
          commit(`setVestingAccount`)
```

**Comment by jbibla** [comment]
i think this should be a yellow message not a red warning.

**Comment by fedekunze** [comment]
it'd be cool to show when the vesting period ends

**Pull request closed without merging**